language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/EnumMemberNodeContext.java

```diff
@@ -32,13 +36,19 @@
  */
 @JavaSPIService("org.ballerinalang.langserver.commons.completion.spi.BallerinaCompletionProvider")
 public class EnumMemberNodeContext extends AbstractCompletionProvider<EnumMemberNode> {
+
     public EnumMemberNodeContext() {
         super(EnumMemberNode.class);
     }
 
     @Override
     public List<LSCompletionItem> getCompletions(BallerinaCompletionContext ctx, EnumMemberNode node)
             throws LSCompletionException {
-        return Collections.emptyList();
+        List<Symbol> visibleSymbols = ctx.visibleSymbols(ctx.getCursorPosition());
+        List<Symbol> filteredSymbols = visibleSymbols.stream()
+                .filter(symbol -> symbol.kind() == SymbolKind.CONSTANT
+                        && ((ConstantSymbol) symbol).broaderTypeDescriptor().typeKind() == TypeDescKind.STRING)
+                .collect(Collectors.toList());
+        return this.getCompletionItemList(filteredSymbols, ctx);
```

Comment:
We need to add the module suggestions as well as the constants coming from the modules,
Eg:
```
enum Test {
    MEMBER1 = <cursor1>
    MEMBER2 = module1:<cursor2>
}
```